docs/core.rst

```diff
@@ -390,16 +390,6 @@ The following exceptions are raised by the Toolkit Core API classes:
     :inherited-members:
     :members:
 
-.. autoclass:: TankFileDoesNotExistError
-    :show-inheritance:
-    :inherited-members:
-    :members:
-
-.. autoclass:: TankInvalidInterpreterLocationError
```

Comment:
What are these things about? Does removing them impact the content of the documentation?

Reply:
Yes, [here](https://developer.shotgridsoftware.com/tk-core/descriptor.html) you can see `TankInvalidAppStoreCredentialsError` repeated.

[Here](https://developer.shotgridsoftware.com/tk-core/core.html#exceptions) you can see `sgtk.TankUnreadableFileError` , `sgtk.TankFileDoesNotExistError`, repeated. In this same page, `sgtk.TankInvalidInterpreterLocationError` is being displayed when it's already defined in the descriptor page. 

I have generated the preview locally, let me know if you want me to share a compressed file with you internally.